dinghy-lib/src/apple/xcode.rs

```diff
@@ -70,6 +70,63 @@ pub fn add_plist_to_app(bundle: &BuildBundle, arch: &str, app_bundle_id: &str) -
     Ok(())
 }
 
+pub fn add_plist_to_tvos_app(bundle: &BuildBundle, arch: &str, app_bundle_id: &str) -> Result<()> {
```

Comment:
Ditto regarding DRY. it is quite difficult to spot the differences between the three `add_plist_to_*_app` functions to understand what's different, and it would be easier to maintain this is we could have the common logic shared.